src/scripts/components/widgets/MainHeaderLogo.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+
+//
+
+type Props = {
+}
+
+//
+
+const MainLogo = styled.h1`
+  display: block;
+  flex: 0 0 ${props => props.theme.sizes ? props.theme.sizes.mainHeaderLogo.width : ''};
+  height: ${props => props.theme.sizes ? props.theme.sizes.mainHeaderLogo.height : ''};
+`
+
+const MainLogoAnchor = styled(Link)`
+  display: block;
+  height: 100%;
+  width: 100%;
+`
+
+const MainLogoSVG = styled.svg`
+  display: block;
+  fill: ${props => props.theme.colors.header.logoFill};
+  height: 100%;
+  width: 100%;
+`
+
+//
```

Comment:
Looks like these `//` comments exist in many of the new components. Can you remove them?